src/nodes/text.ts

```diff
@@ -123,8 +150,11 @@ export class TextNode extends GraphicsNode {
         if (xmlSpace === 'default') {
           if (i === 0) {
             trimmedText = trimLeft(trimmedText)
+            if (originalText.match(/^[(\n\r)|(\t)| ]/)) {
+              lengthAdjustment = 0
+            }
           }
-          if (i === tSpanCount - 1) {
+          if (i === this.element.childNodes.length - 1) {
```

Comment:
I think this line was originally a bug as `this.element.childElementCount` (`tSpanCount`) doesn't always equal `this.element.childNodes.length`. Given this loop is over `this.element.childNodes`, it makes sense to use the latter when checking for the last element in the array!